filebeat/module/elasticsearch/gc/_meta/fields.yml

```diff
@@ -1,20 +1,110 @@
 - name: gc
   type: group
   description: >
+    GC fileset fields.
   fields:
-    - name: relative_process_timestamp_secs
+    - name: phase
+      type: group
+      description: >
+        Fields specific to GC phase.
+      fields:
+        - name: name
+          type: keyword
+          description: >
+            Name of the GC collection phase.
+        - name: duration_sec
+          type: float
+          description: >
+            Collection phase duration according to the Java virtual machine.
+        - name: scrub_symbol_table_time_sec
```

Comment:
Same comment as above and also for the following fields.

General note: I would prefer not to have sec and float but instead ms and a long. But not sure if we can make that happen here. 

What should be the accuracy of these fields?